docs/conops.md

```diff
@@ -0,0 +1,582 @@
+
+## 3.0 Description of Envisioned System
+
+## 3.1 Needs, Goals and Objectives of Envisioned System
+Each service in this architecture is associated with a set of requirements.
+
+Each service has its own set of documents which should be referred to for more detailed information. These documents include:
+- Concept of Operations (CONOPS)
+- Software Design Document (SDD)
+- Interface Control Document (ICD)
+- Verification and Validation (V&V) Document
+
+A set of user stories for rideshare and cargo operations is currently tracked on the [Arrow Google Drive](https://docs.google.com/spreadsheets/d/1Ad238NAEj6QUzgsjPTRRFJy6NiQVQt2e7affwVVDAFo/edit?usp=sharing).
+
+## 3.2 Overview of System and Key Elements
+
+### 3.2.1 Provider of Services (PSU) Elements
+
+These services enable UAM operators to access airspace legally, safely, and securely.
+
+<center> 
+
+Service | Responsibilities
+--- | ---
+`svc-scheduler` | Dynamic airspace management<br>Vertiport scheduling<br>Flight plan bookkeeping<br>Fleet routing optimization
+`svc-discovery` | Interaction with other PSUs<br>Adheres to standardized PSU interfaces
+`svc-compliance` | Notifications, flight restrictions, authorization approvals from local authority<br>Automated dispatch of flight plans and release requests to local authority<br>Complies with regulatory file formats such as FIDXP<sup>*</sup>
+`svc-guidance` | Noncompulsory flight path guidance<br>Importance increases with number of aerial actors
+
+<sup>*</sup> FAA-Industry Data Exchange Protocol
+
+</center> 
+
+### 3.2.2 Supplemental Data and Service Provider (SDSP) Elements
+
+These elements provide *supplemental* (non-essential) services for UAM operations.
+
+<center> 
+
+Service | Responsibilities
+--- | ---
+`svc-storage` | Abstracted storage interface for the other services<br>Logs, maintenance records, certification records, and so on
+`svc-telemetry` | Receives and stores live vehicle telemetry<sup>*</sup><br>Rebroadcasts to ground control/air traffic control tools and other listeners
+`svc-upkeep` | Monitor expiration of vehicle and component certifications<br>Preemptively schedule flights for maintenance
+
+</center>
+
+<sup>*</sup> May qualify as a PSU element.
+### 3.2.3 Rideshare and Cargo Services
+
+These are client-facing services exposing an API. They handle transactions with customers and are not essential to the operation of the PSU.
+
+<center> 
+
+Service | Responsibilities
+--- | ---
+`svc-rideshare` | The public API for passengers to request rides through Arrow Services 
+`svc-cargo` | The public API for clients to request cargo transport through Arrow Services
+`svc-charter`| The public API for individuals filing charters for specific aircraft.
+`svc-payment` | Payment processing, accepting traditional payments and cryptocurrencies.
+`svc-contact` | This service has sole access to customer information, the interface to issue notifications to a client
+
```

Comment:
add `svc-pricing` as the service used to request pricing quotes during flight planning

Reply:
https://github.com/Arrow-air/se-services/pull/2/commits/fd87e1512c5648b706fd9161d66063f763adc043